src/Microsoft.DocAsCode.Build.Engine/TemplateModelTransformer.cs

```diff
@@ -176,6 +177,32 @@ internal ManifestItem Transform(InternalManifestItem item)
             return appended;
         }
 
+        private IDictionary<string, object> ResolveBreadCrumb(IDictionary<string, object> model)
```

Comment:
And we should let template to decide which metadata should be treated as path.

---
In reply to: [72409208](https://github.com/dotnet/docfx/pull/538#discussion_r72409208) [](ancestors = 72409208)